test/pid_tests.cpp

```diff
@@ -121,6 +121,31 @@ TEST(ParameterTest, negativeIntegrationAntiwindupTest)
   EXPECT_EQ(1.0, cmd);
 }
 
+TEST(ParameterTest, negativeAsymmetricIntegrationAntiwindupTest)
```

Comment:
IMHO we don't need another test. Just update the values in negativeIntegrationAntiwindupTest :)